mkdocs.yml

```diff
@@ -249,4 +261,4 @@ validation:
   unrecognized_links: warn
   anchors: warn
 hooks:
-  - docs/hooks/d2-svg-file.py
+  - docs/scripts/d2-svg-file.py
```

Comment:
Tiniest of nits, I think of `scripts` as things that are run by people, IMO `hooks` was a better name.

Reply:
I did this to reduce the number of directories containing tiny mkdocs scripts that do things.